.github/workflows/test.yml

```diff
@@ -167,6 +170,11 @@ jobs:
           # Galactic Geochelone (May 2021 - November 2022)
           - docker_image: rostooling/setup-ros-docker:ubuntu-focal-ros-galactic-ros-base-latest
             ros_distribution: galactic
+
+          # Humble Hawksbill (May 2022 - May 2027)
```

Comment:
We seem to be missing Rolling Ridley here..

Reply:
Indeed, I added some more jobs for rolling: 92d3580e716ebdc4d73b78d0cf1b1570f7c235c8